connexion/options.py

```diff
@@ -44,11 +48,10 @@ def openapi_spec_available(self):
         # type: () -> bool
         """
         Whether to make available the OpenAPI Specification under
-        `openapi_console_ui_path`/swagger.json path.
+        `openapi_spec_path`.
 
         Default: True
         """
-        # NOTE: Under OpenAPI v3 this should change to "/openapi.json"
```

Comment:
Thoughts on renaming the option? Backwards compatibility?

Reply:
We can rename it. Maybe using `swagger_json` as a fallback (with a deprecation warning) if `openapi_json` is not set.

Reply:
I'm signing off for the night - If you could please take a look at https://github.com/zalando/connexion/pull/656 also, that would be great - It should be a quick bugfix. Have a good evening.